pymc3/distributions/multivariate.py

```diff
@@ -248,40 +253,35 @@ def random(self, point=None, size=None):
                 except TypeError:
                     size = (size,)
 
-        if self._cov_type == 'cov':
+        if self._cov_type == "cov":
             mu, cov = draw_values([self.mu, self.cov], point=point, size=size)
             if mu.shape[-1] != cov.shape[-1]:
                 raise ValueError("Shapes for mu and cov don't match")
 
             try:
-                dist = stats.multivariate_normal(
-                    mean=mu, cov=cov, allow_singular=True)
+                dist = stats.multivariate_normal(mean=None, cov=cov, allow_singular=True)
```

Comment:
This may fix the nans seen in #3758 but once someone makes `cov` an RV, the same shape issues will pop up again. The exception will be silenced here and the user will only get back nans (which I think is wrong).